src/test/java/unit/BqTableReferenced.java

```diff
@@ -429,20 +432,71 @@ void updateMultipleOrNoProperties() throws IOException {
     assertEquals(newDescription, updateDataTable.description);
 
     // `terra resources describe --name=$newName`
-    UFBqDataTable describeDataTable =
+    UFBqTable describeDataTable =
         TestCommand.runAndParseCommandExpectSuccess(
-            UFBqDataTable.class, "resource", "describe", "--name=" + newName);
+            UFBqTable.class, "resource", "describe", "--name=" + newName);
     assertEquals(newDescription, describeDataTable.description);
   }
 
+  @Test
+  @DisplayName("referenced dataset with no access does not fail the describe command")
+  void numRowsForReferencedWithNoAccess() throws IOException {
+    workspaceCreator.login();
+
+    // `terra workspace set --id=$id`
+    TestCommand.runCommandExpectSuccess("workspace", "set", "--id=" + getWorkspaceId());
+
+    // `terra resource add-ref bq-table --name=$name --project-id=$projectId
+    // --dataset-id=$datasetId`
+    String name = "numRowsForReferencedWithNoAccess";
+    TestCommand.runCommandExpectSuccess(
+        "resource",
+        "add-ref",
+        "bq-table",
+        "--name=" + name,
+        "--project-id=" + externalDataset.getProjectId(),
+        "--dataset-id=" + externalDataset.getDatasetId(),
+        "--table-id=" + externalDataTableName);
+
+    // `terra resource describe --name=$name`
+    UFBqTable describeDataTable =
+        TestCommand.runAndParseCommandExpectSuccess(
+            UFBqTable.class, "resource", "describe", "--name=" + name);
+
+    // the external dataset created in the beforeall method should have 1 table in it, but the
+    // sharee user doesn't have read access to the table so they can't know that
+    assertEquals(
+        BigInteger.ZERO,
+        describeDataTable.numRows,
+        "referenced dataset with no access contains NULL numRows");
```

Comment:
The actual assertion is fine, but I don't think this comment and assertion message make sense. Here you're checking that the CLI reports zero rows for a user that does have access.